src/electionguard/encrypt.py

```diff
@@ -35,18 +35,39 @@ class EncryptionDevice(Serializable):
     """
 
     uuid: int
+    """Unique identifier for device"""
+
+    session_id: str
+    """Used to identify session and protect the timestamp"""
+
+    launch_code: int
+    """Election initialization value"""
+
     location: str
+    """Arbitary string to designate the location of device"""
 
-    def __init__(self, location: str) -> None:
+    def __init__(
+        self,
+        session_id: str,
+        launch_code: int,
+        location: str,
+    ) -> None:
         self.uuid = generate_device_uuid()
```

Comment:
im passing this as a constructor parameter... im not sure which way is the better way to do it but we should be consistent.

Reply:
if it's a construction parameter it will have to be the last parameter. It's labeled as the first object in the hash.